assets/components/cards/render/WireListCard.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import MoreNewsButton from './MoreNewsButton';
+import classNames from 'classnames';
+import {characterCount, wordCount} from 'utils';
+import WireListItemIcons from 'wire/components/WireListItemIcons';
+import {FieldComponents} from 'wire/components/fields';
+import {Embargo} from 'wire/components/fields/Embargo';
+import {UrgencyItemBorder, UrgencyLabel} from 'wire/components/fields/UrgencyLabel';
+import {isKilled, getCaption, getImageForList} from 'wire/utils';
+import {connect} from 'react-redux';
+import {listConfigSelector} from 'ui/selectors';
+import {DEFAULT_COMPACT_META_FIELDS} from 'wire/components/WireListItem';
+import {IArticle, IListConfig} from 'interfaces';
+import {filterGroupsToLabelMap} from 'search/selectors';
+import {setActive, openItem} from 'wire/actions';
+
+interface IOwnProps {
+    item: IArticle;
+}
+
+interface IPropsReduxStore {
+    listConfig: IListConfig;
+    filterGroupLabels: any;
+    dispatch: any;
+}
+
+type IPropsCombined = IPropsReduxStore & IOwnProps;
+
+class WireListPanel extends React.Component<IPropsCombined> {
+    wordCount: number;
+    characterCount: number;
+
+    constructor(props: IPropsCombined) {
+        super(props);
+
+        this.wordCount = wordCount(props.item);
+        this.characterCount = characterCount(props.item);
+    }
+
+    render(): React.ReactNode {
+        const cardClassName = classNames('wire-articles__item-wrap col-12 wire-item');
+        const wrapClassName = classNames('wire-articles__item wire-articles__item--wire wire-articles__item--list');
+        const listImage = getImageForList(this.props.item);
+        const compactFields = this.props.listConfig.compact_metadata_fields || DEFAULT_COMPACT_META_FIELDS;
+        const {item, listConfig} = this.props;
+        const isExtended = false;
+
+        const onItemClick = () => {
```

Comment:
I can see that other Card components use the [ICardProps](https://github.com/superdesk/newsroom-core/blob/develop/assets/components/cards/utils.ts#L40) interface, which provides an `openItem` prop (among others).

It would make it easier to control the functionality if these props were used instead of directing importing/executing the wire actions. This flexibility would be required when reusing these Card components for the public dashboard